src/Illuminate/Cache/CacheServiceProvider.php

```diff
@@ -22,6 +23,10 @@ public function register()
             return $app['cache']->driver();
         });
 
+        $this->app->singleton('cache.psr6', function ($app) {
+            return new Psr16Adapter($app['cache.store']);
```

Comment:
Yes, there's no typo here. The class named `Psr16Adapter` takes a PSR-16 implementation and turns it into a PSR-6 one. :)

Reply:
Since is this new, we could probably just bind the singleton diredctly to the PSR interface instead of having an alias.

Reply:
Ok, though I'd like to bind also to the other 2 classes that I've included. Should I directly alias this within the service provider? Let me know your preferred style. :)